molior-deploy

```diff
@@ -502,103 +511,109 @@ setup_deployment()
   log " * started: `date -R`"
   log " * logfile: $WORK_DIR/output.log"
 
-  if [ -z "$MOLIOR_PROJECTSOURCES_FILE" ]; then
-    log_info "Getting deployment information ..."
-    SOURCESURL=$MOLIOR_SERVER/api/projectsources/$PROJECT/$VERSION
-    APT_SOURCES_ORIG=`wget -q -O- $SOURCESURL`
-    if [ -z "$APT_SOURCES_ORIG" ]; then
-      log_error "Error downloading $SOURCESURL"
+  if [ -z "$BASE_LAYER" ]; then
+    if [ -z "$MOLIOR_PROJECTSOURCES_FILE" ]; then
+      log_info "Getting deployment information ..."
+      SOURCESURL=$MOLIOR_SERVER/api/projectsources/$PROJECT/$VERSION
+      APT_SOURCES_ORIG=`wget -q -O- $SOURCESURL`
+      if [ -z "$APT_SOURCES_ORIG" ]; then
```

Comment:
why not checking exit status?

Reply:
I think it is cleaner if I anyway know that it will fail, I check it beforehand and the don't do it at all. :)